frontend/src/components/details/SelectVariableComponent.jsx

```diff
@@ -96,7 +87,7 @@ class SelectVariableComponent extends Component {
                         </div>
                     </div>
 
-                    <button
+                    {/* <button
```

Comment:
If you don't use it, please remove it, instead of commenting it out. We can always get back code from earlier commits if required, so it's not lost forever ☺